nemo_aligner/data/nlp/datasets.py

```diff
@@ -14,12 +14,14 @@
 
 """Custom datasets for RLHF training"""
 
+import math
 import os
 from typing import Dict, List
 
 import numpy as np
 import scipy
 import torch
+from megatron.core import parallel_state
```

Comment:
could you switch to using `nemo_aligner.utils.parallel_state` to match the rest of the repo?